src/test/java/com/amazon/opendistroforelasticsearch/jdbc/PreparedStatementTests.java

```diff
@@ -1,32 +1,30 @@
 package com.amazon.opendistroforelasticsearch.jdbc;
 
-import java.io.IOException;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.Collections;
-
 import com.amazon.opendistroforelasticsearch.jdbc.config.ConnectionConfig;
 import com.amazon.opendistroforelasticsearch.jdbc.logging.NoOpLogger;
-import com.amazon.opendistroforelasticsearch.jdbc.protocol.ConnectionResponse;
-import com.amazon.opendistroforelasticsearch.jdbc.protocol.JdbcQueryRequest;
-import com.amazon.opendistroforelasticsearch.jdbc.protocol.Protocol;
-import com.amazon.opendistroforelasticsearch.jdbc.protocol.ProtocolFactory;
-import com.amazon.opendistroforelasticsearch.jdbc.protocol.QueryRequest;
-import com.amazon.opendistroforelasticsearch.jdbc.protocol.QueryResponse;
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.*;
```

Comment:
Please avoid using wildcard import. You can enforce single import by configuring your IDE. Instructions for Intellij IDEA: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports.